### ISSUE TITLE
Could not find formatter 'node_modules/eslint-friendly-formatter/index.js'. (Doesn't work with a fresh node_modules)

### ISSUE THREAD

**Issue by MoOx** [royriojas/eslint-friendly-formatter#3]
I just get an issue on Travis CI about complaining about not finding this formatter.
So I cleaned my local node_modules and reinstalled from my pakcage.json and now I get this 

```console
$ eslint -f node_modules/eslint-friendly-formatter/index.js --ext .js,.jsx,.es .

Could not find formatter 'node_modules/eslint-friendly-formatter/index.js'.

npm ERR! Darwin 14.3.0
npm ERR! argv "/Users/MoOx/.nvm/versions/io.js/v2.0.1/bin/iojs" "/Users/MoOx/.nvm/version
js/v2.0.1/bin/npm" "run" "lint"
npm ERR! node v2.0.1
npm ERR! npm  v2.9.0
npm ERR! code ELIFECYCLE
npm ERR! putaindecode.fr@0.0.0 lint: `eslint -f node_modules/eslint-friendly-formatter/in
s --ext .js,.jsx,.es .`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the putaindecode.fr@0.0.0 lint script 'eslint -f node_modules/eslint-f
ly-formatter/index.js --ext .js,.jsx,.es .'.
npm ERR! This is most likely a problem with the putaindecode.fr package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     eslint -f node_modules/eslint-friendly-formatter/index.js --ext .js,.jsx,.es
npm ERR! You can get their info via:
npm ERR!     npm owner ls putaindecode.fr
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/MoOx/Sync/Development/putaindecode/putaindecode.fr/npm-debug.log

~/Sync/Development/putaindecode/putaindecode.fr feat.hero*    ⇡
❯❯ ls node_modules/eslint-friendly-formatter/index.js
node_modules/eslint-friendly-formatter/index.js
```
As you can see with the `ls` above, the file exist (and have the expected content).
eslint cli still have this option so not sure what is going on.

Any idea ?

**Comment by royriojas** [comment]
mmm, sorry my bad. I found the issue in the meantime you can use the previous version. Basically a file was not included inside the package and it is breaking things. Thanks for the report

**Comment by royriojas** [comment]
Closed in v1.0.7.
